hnn_core/tests/test_dipole.py

```diff
@@ -61,7 +62,33 @@ def test_dipole_simulation():
         simulate_dipole(net, n_trials=0)
     with pytest.raises(TypeError, match="record_vsoma must be bool, got int"):
         simulate_dipole(net, n_trials=1, record_vsoma=0)
+    with pytest.raises(TypeError, match="record_isoma must be bool, got int"):
+        simulate_dipole(net, n_trials=1, record_vsoma=False, record_isoma=0)
 
-    simulate_dipole(net, n_trials=2, record_vsoma=True)
-    assert len(net.spikes.vsoma) == 2
-    assert len(net.spikes.vsoma[0]) == 24
+    with MPIBackend(n_procs=None, mpi_cmd='mpiexec'):
+        simulate_dipole(net, n_trials=2, record_vsoma=True, record_isoma=True)
+        assert len(net.cell_response.vsoma) == 2
+        assert len(net.cell_response.isoma) == 2
+        n_times = np.arange(0., params['tstop'] + params['dt'],
+                            params['dt']).size
+        assert len(net.cell_response.vsoma[0][8]) == n_times
```

Comment:
what's 8?

Reply:
It's just the gid `vsoma` is recorded from. I'll assign explicit variables along with `n_trials` above.